go/runtime/history/indexer.go

```diff
@@ -15,6 +15,35 @@ import (
 	roothash "github.com/oasisprotocol/oasis-core/go/roothash/api"
 )
 
+const (
+	StatusStarted    = "started"
```

Comment:
I would make this private, as otherwise someone can use `history.StatusStarted` and has no idea to which struct this status belongs (to history?). And public constants should also have comments, if we keep as it is.